kiva/agg/tests/test_join_stroke_path.py

```diff
@@ -120,17 +119,21 @@ def test_antialias_miter(self):
 
         gc = self.helper(antialias, width, line_cap, line_join)
 
-        actual = gc.bmp_array[:,:,0]
-        desired = array([[255, 255, 255, 255, 255, 255, 255, 255, 255, 255],
-                         [255, 255, 255, 255, 255, 127,   0,   0, 127, 255],
-                         [255, 255, 255, 255, 255, 127,   0,   0, 127, 255],
-                         [255, 255, 255, 255, 255, 127,   0,   0, 127, 255],
-                         [255, 255, 255, 255, 255, 127,   0,   0, 127, 255],
-                         [255, 127, 127, 127, 127,  63,   0,   0, 127, 255],
-                         [255,   0,   0,   0,   0,   0,   0,   0, 127, 255],
-                         [255,   0,   0,   0,   0,   0,   0,   0, 127, 255],
-                         [255, 127, 127, 127, 127, 127, 127, 127, 191, 255],
-                         [255, 255, 255, 255, 255, 255, 255, 255, 255, 255]])
```

Comment:
hehehehe. looks like black ruined the nice clean formatting that denoted what was being drawn.

Reply:
We can undo that.